tests/integration/navigation/navigation_base.js

```diff
@@ -90,8 +90,8 @@ asyncTest( "can navigate between internal and external pages", function( assert
 				report: "navigate from a page in a non-base directory to an internal page"
 			} );
 
-			// Try calling changePage() directly with a relative path.
-			$.mobile.changePage( "base-page-1.html" );
+			// Try calling change() directly with a relative path.
+			$( ".ui-pagecontainer" ).pagecontainer( "change", "base-page-1.html" );
```

Comment:
I guess this is OK for testing purposes since we control the content and we know there is only one pagecontainer. Still, maybe $.mobile.pagecontainers.active is in order?